client/client.go

```diff
@@ -514,6 +514,16 @@ type HttpsClient struct {
 	SSLEngine   string
 }
 
+// Security structure holds the configuration for the client
+// Added for MEN-3924 in order to provide a way to specify PKI params
+// outside HttpsClient.
+// NOTE: Careful when changing this, the struct is exposed directly in the
+// 'mender.conf' file.
+type Security struct {
+	PKIPrivateKey string
+	SSLEngine     string
+}
```

Comment:
Remember to update [the documentation](https://docs.mender.io/client-installation/configuration-file/configuration-options) as well.

Reply:
yes Sir! https://github.com/mendersoftware/mender-docs/pull/1209

Reply:
Is it too late to have a slight discussion around the wording?
I'm having doubts about the `security` name?

Reply:
Got a better suggestion?

Reply:
You guys decide on the quality, I'm just going to name suggestions:
* `PrivateKey`?
* `PKI`?
* `HardwareSecurity`
* `HardwareKey`
* `SSLEngineKey`
* `PluginKey`

Reply:
of course it is too late! ;-) @oleorhagen right now it is like that:
```
    "Security": {
        "PKIPrivateKey": "...",
        "SSLEngine": "..."
    },
```
What exactly do you suggest to change?

Reply:
The `Security` subfield in the `mender.conf` file. Security sounds broad to me.

Reply:
I agree but I did that on purpose, I wanted to use `Security` for the upcoming settings API-related and related to the new features we are planning. Those should fall nicely into that section.

Reply:
Alright, makes sense then.